### DISCUSSION TITLE
Different Layout and Page on home route based on role

### DISCUSSION THREAD

**Discussion by surnr** [vercel/next.js#61333] [App Router]
I have 3 different templates which should be shown in the home (i.e. "/") route.
Let's say I have 3 different group routes (group1), (group2), and (group3) each having their template page and layout files.
Based on role1, role2, role3 render the route group group1, group2, group 3 respectively.

How can we achieve this, Is there any better or easy way to do this?
Also, I don't want to render the pages in 3 different routes ("/route1", "/route2", "/route3"), I want to render on the home route ("/") only.

Thanks for the help peeps.

**Comment by nirvikpurkait** [comment]
If you want to render multiple route group on a single route i.e:  `/`, you have to create multiple RootLayout, to create multiple RootLayout, You can check-out [this](https://nextjs.org/docs/app/building-your-application/routing/route-groups#creating-multiple-root-layouts) docs

**Comment by surnr** [comment]
- Routes that include a route group should not resolve to the same URL
   path as other routes. For example, since route groups don't affect URL
   structure, (marketing)/about/page.js and (shop)/about/page.js would both
   resolve to /about and cause an error.

**Comment by nirvikpurkait** [comment]
Well I can think of a solution that you can make use of [`parallel route`](https://nextjs.org/docs/app/building-your-application/routing/parallel-routes) to render different `page.tsx` in the same route.
